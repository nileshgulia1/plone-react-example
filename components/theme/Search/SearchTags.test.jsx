import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

import SearchTags from './SearchTags';

const mockStore = configureStore();

describe('SearchTags', () => {
  it('renders a search tags component', () => {
    const store = mockStore({
      vocabularies: {
        'plone.app.vocabularies.Keywords': {
          terms: [{ title: 'Tag 1' }, { title: 'Tag 2' }],
        },
      },
    });
    const component = renderer.create(
      <Provider store={store}>
        <SearchTags />
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
