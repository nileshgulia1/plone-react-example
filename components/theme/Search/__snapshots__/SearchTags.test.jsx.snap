// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchTags renders a search tags component 1`] = `
<div>
  <a
    className="ui label"
    onClick={[Function]}
    style={Object {}}
  >
    Tag 1
  </a>
  <a
    className="ui label"
    onClick={[Function]}
    style={Object {}}
  >
    Tag 2
  </a>
</div>
`;
